internal/pkg/redis/client.go

```diff
@@ -250,18 +252,20 @@ func createRedisClient(
 
 func convertToRedisTopicScheme(topic string) string {
```

Comment:
Please add unit test for this function with various inputs/outputs

Reply:
Added new unit tests.